.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,5 +3,8 @@
 
 Only make Pull Requests against other branches (e.g. `2.1`) if your changes only apply to that specific version of Godot.
 
-All pull requests for Godot 3 should usually go into `master`.
+All pull requests for the latest version of Godot should usually go into `master`.
```

Comment:
I'm not sure this sentence is useful at all since it's redundant with the first one.